extras/bindings/c/include/precice/SolverInterfaceC.h

```diff
@@ -450,6 +450,31 @@ const char *precicec_actionWriteIterationCheckpoint();
 // @brief Name of action for reading iteration checkpoint.
 const char *precicec_actionReadIterationCheckpoint();
 
+///@}
+
+/** @name Experimental Data Access
+ * These API functions are \b experimental and may change in future versions.
+ */
+///@{
+
+/**
+ * @brief Have a look at the c++ API documentation
```

Comment:
You can directly link to the C++ entry.
Please check if this generates the correct link though.

```suggestion
 * @see precice::SolverInterface::setMeshAccessRegion()
 ```

Same below

Reply:
What I have seen in other doxygen code is also the directive `@copydoc`. Have you ever considered the usage here?

Reply:
I wasn't aware of that!
You could use it, as long as the documented parameters are identical.